### PR TITLE
new yield curve - aspen hardfork

### PR DESCRIPTION
**Pull request by JakubFornadel**
## Purpose

- implemented new dynamic yield curve
- added rpc calls taraxa_totalSupply & taraxa_yield

 evm pr:
https://github.com/Taraxa-project/taraxa-evm/pull/199

### CODE REVIEW

File: libraries/core_libs/network/rpc/TaraxaFace.h [hunk 1]
```diff
@@ -44,6 +44,12 @@ class TaraxaFace : public ServerInterface<TaraxaFace> {
     this->bindAndAddMethod(jsonrpc::Procedure("taraxa_pbftBlockHashByPeriod", jsonrpc::PARAMS_BY_POSITION,
                                               jsonrpc::JSON_STRING, "param1", jsonrpc::JSON_STRING, NULL),
                            &taraxa::net::TaraxaFace::taraxa_pbftBlockHashByPeriodI);
+    this->bindAndAddMethod(
+        jsonrpc::Procedure("taraxa_yield", jsonrpc::PARAMS_BY_POSITION, jsonrpc::JSON_STRING, "param1", NULL),
```
[the remark this answers is not in the document]
Comment (JakubFornadel) [reply libraries/core_libs/network/rpc/TaraxaFace.h:48]: hmm but taraxa_pbftBlockHashByPeriod for example uses JSON_STRING for period ?

[the remark this answers is not in the document]
Comment (JakubFornadel) [reply libraries/core_libs/network/rpc/TaraxaFace.h:48]: this actually works with both decimal and hex string value

[the remark this answers is not in the document]
Comment (JakubFornadel) [reply libraries/core_libs/network/rpc/TaraxaFace.h:48]: yes I tried it on localnet and it works

[the remark this answers is not in the document]
Comment (JakubFornadel) [reply libraries/core_libs/network/rpc/TaraxaFace.h]: changed it to JSON_ANY.

**Review by mfrankovi — APPROVED** [review]

**Pull request closed without merging**